llama_toolchain/core/datatypes.py

```diff
@@ -188,3 +188,19 @@ class PackageConfig(BaseModel):
 the dependencies of these providers as well.
 """,
     )
+
+
+@json_schema_type
+class BuildConfig(BaseModel):
+    name: str
+    distribution: str = Field(
```

Comment:
nit: rename to `distribution_type` for consistency

Reply:
Renamed all to `distribution` to avoid the hassle of typing _type.